nengo/utils/least_squares_solvers.py

```diff
@@ -253,7 +270,11 @@ class RandomizedSVD(LeastSquaresSolver):
     ``sklearn.utils.extmath.randomized_svd`` for details about the parameters.
     """
 
+    n_components = IntParam('n_components', low=1)
+    n_oversamples = IntParam('n_oversamples', low=0)
+
```

Comment:
Does `DictParam` have a proper hashing function right now? Maybe we should just add all the possible parameters manually. There aren't too many of them.